Sources/wkhtmltopdf/Document.swift

```diff
@@ -1,23 +1,23 @@
 public class Document {
 
-  // This may need changing across different platforms and deployments.
-  static var zoom: String = "1.3"
+    // This may need changing across different platforms and deployments.
+    static var zoom: String = "1.3"
```

Comment:
I _think_ static vars are no longer a good idea with Vapor 3. I think it's reasonable to either make it a simple var which the user sets on each Document, or to make it an argument to `generatePDF()`.